manifests/repo/debian.pp

```diff
@@ -2,66 +2,30 @@
 #
 # === Parameters
 #
-# [*location*]
-#   Location of the apt repository
-#
-# [*release*]
-#   Release of the apt repository
-#
-# [*repos*]
-#   Apt repository names
-#
 # [*include_src*]
 #   Add source source repository
 #
 # [*key*]
 #   Public key in apt::key format
 #
-# [*dotdeb*]
-#   Enable special dotdeb handling
-#
-# [*sury*]
-#   Enable special sury handling
-#
 class php::repo::debian (
-  $location     = 'https://packages.dotdeb.org',
```

Comment:
definitly against being able to set a custom repo for debian. removing this is a nogo for me.

Reply:
fair point i ll rework

Reply:
done